pkg/backends/healthcheck/healthcheck_test.go

```diff
@@ -62,7 +62,7 @@ func TestRegister(t *testing.T) {
 	logger.SetLogger(testLogger)
 	hc := New().(*healthChecker)
 	o := ho.New()
-	o.IntervalMS = 500
+	o.Interval = 500
```

Comment:
while this (and the 3 similar operations below)  probably works, the 1:1 would be `500 * time.Millisecond`